conjure-go-client/httpclient/refreshable_config.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2021 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package httpclient
+
+import (
+	"net/http"
+	"time"
+)
+
+func (b *httpClientBuilder) handleIdleConnUpdate(c *clientImpl) {
+	b.IdleConnTimeout.SubscribeToDuration(func(v time.Duration) {
+		if unwrappedTransport := unwrapTransport(c.client.Transport); unwrappedTransport != nil {
+			unwrappedTransport.IdleConnTimeout = v
+			c.client.Transport = unwrappedTransport
```

Comment:
Are these actually safe to assign directly like this given we read this value in the `doOnce` function?